src/scss/login.module.scss

```diff
@@ -279,11 +284,379 @@ $white: #fff;
 }
 
 .button--github {
-  color: #333;
+  color: #3c4b64;
   line-height: 46px;
   margin-bottom: 3rem;
 
   &:hover {
     color: $tiber;
   }
 }
+.btnSlideEffect:hover,
+.btnSlideEffect:focus {  
+  animation: pulse 1s;
+  background-color: #3c4b64;
+  color:#ebedef;
+  box-shadow: 0 0 0 2em rgba(#3c4b64,0);
+}
+
+// /////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Landing Page Sections CSS
+// Section Container
+.slide{
+  height:100%;
+  position: relative;
+  padding:1em;
+  font-size:14px;
+  margin-top:8em;
+  }
+
+ 
+  // Feature Section
+  .Features
+	{
+		display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap:0em;
+    margin-top:7em;
+    height:100%;
+		
+  }
+
+  .Features:nth-child(1)
+    {
+        margin-top:0;
+    } 
+
+  .TextContainer{
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding:2em;
+    box-shadow:0px 2px 3px 0px rgba(0,0,0,.25);
+    border-radius: 2px;;
+    position: relative;
+    gap:3em;
+    text-align: left;
+  }
+
+  .TextContainer >h2{
+    text-align:center;
+  }
+  
+  .Features > .TextContainer ,.AboutUs > .TextContainer
+  {
+    padding-right:2em;
+    height:fit-content;
+    align-self: center;
+    flex-basis:50%;
+    box-shadow:none;
+    
+  }
+
+  .GifCard{
+    
+    height:100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    
+  }
+  .GifCard > img{
+    width:100%;
+    
+    box-shadow:  0 0 5px 0 rgba(0,0,0,0.25);
+  }
+
+  .IntroText h1{
+    margin-bottom: 1em;
+    display: inline-block;
+    position: relative;
+  }
+
+  .IntroText li, .IntroText p{
+    line-height: 1.5em;
+  }
+
+  // AboutUs Section
+
+  .AboutUs > .GifCard > img{
```

Comment:
There is no About Us section on the page so let's make this `about` or something similar. Same for the comment on 383.